ParisBNF/abb/BNFabb20.xml

```diff
@@ -41,8 +41,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         
                             
                             <msItem xml:id="ms_i1.1">                              
-                            <locus from="3r" to="4v"/>
+                            <locus from="3ra" to="4vb"/>
                                 <title type="complete" ref="LIT3171Commen#Summary">Index</title>
```

Comment:
here again the title will be printed only as "Index"

Reply:
Oh, sorry. Corrected.